src/manage-authors/add-coauthor.js

```diff
@@ -0,0 +1,17 @@
+const { gitAuthors } = require('../git-authors');
+
+async function addCoauthor(args) {
+  const coauthors = gitAuthors();
+  const authorList = await coauthors.read();
+  if (Object.prototype.hasOwnProperty.call(authorList.coauthors, args[0])) {
```

Comment:
To make this more readable I would assign `args[0]` to a variable like `key` .
Then in the if use `if(key in authorList.coauthors)`.